operator/src/main/resources/scripts/startServer.sh

```diff
@@ -188,12 +147,41 @@ checkEnv \
   AS_SERVICE_NAME || exitOrLoop
 
 trace "LOG_HOME=${LOG_HOME}"
+trace "DATA_HOME=${DATA_HOME}"
 trace "SERVER_OUT_IN_POD_LOG=${SERVER_OUT_IN_POD_LOG}"
 trace "USER_MEM_ARGS=${USER_MEM_ARGS}"
 trace "JAVA_OPTIONS=${JAVA_OPTIONS}"
+trace "KEEP_DEFAULT_DATA_HOME=${KEEP_DEFAULT_DATA_HOME}"
+trace "EXPERIMENTAL_LINK_SERVER_DEFAULT_DATA_DIR=${EXPERIMENTAL_LINK_SERVER_DEFAULT_DATA_DIR}"
+
+# The following is experimental code that handles the specific case of services that don't provide a configurable way to
+# control the location of their persistent file data.  For example, web applications can configure file-based
+# session persistence where the default persistent file store location is automatically created in the
+# <server-name>\data\store\default directory.
```

Comment:
Minor comment:  The above comment is out-of-place it should be moved/merged into into line 176 comment.